app/src/main/java/org/simple/clinic/scheduleappointment/ScheduleAppointmentModel.kt

```diff
@@ -13,7 +14,8 @@ data class ScheduleAppointmentModel(
     val patientUuid: UUID,
     val potentialAppointmentDates: List<PotentialAppointmentDate>,
     val selectedAppointmentDate: PotentialAppointmentDate?,
-    val appointmentFacility: Facility?
+    val appointmentFacility: Facility?,
+    val doneButtonState : ButtonState?
```

Comment:
We can make it non-null and provide the default state to be `Enabled` in `create`, WDYT?

Reply:
This makes sense. I'll do it